### PR TITLE
add 22H2 and 22621

### PR DESCRIPTION
**Pull request by sethbrasile**
Add 22H2 and 22621 to `Get-WindowsVersion` and `Get-DesktopWindowsVersionComparison`

### CODE REVIEW

File: Function.Get-DesktopWindowsVersionComparison.ps1 [hunk 1]
```diff
@@ -44,7 +44,6 @@ If ($winIsLessThan20H2.Result) {
 Try {
   # Oddly, this command works to enable TLS12 on even Powershellv2 when it shows as unavailable. This also still works for Win8+
   [Net.ServicePointManager]::SecurityProtocol = [Enum]::ToObject([Net.SecurityProtocolType], 3072)
-  $outputLog += "Successfully enabled TLS1.2 to ensure successful file downloads."
```
Comment (sethbrasile) [inline Function.Get-DesktopWindowsVersionComparison.ps1:47]: removed this b/c it keeps ending up in the output of all consuming scripts and it doesn't make sense

File: Function.Get-DesktopWindowsVersionComparison.ps1 [hunk 2]
```diff
@@ -55,8 +54,9 @@ Try {
   }
 }
 
+# TODO: replace with master URL before merge
```
Comment (sethbrasile) [inline Function.Get-DesktopWindowsVersionComparison.ps1:57]: Note that this change (and also on line 59) only exists to facilitate testing this branch and will be reverted before merge

File: Function.Get-WindowsVersion.ps1 [hunk 3]
```diff
@@ -9,7 +9,7 @@
         Build = $Null
         Version = $Null
         Arch = $osArch
-        OrderOfWindowsVersions = @('1507','1511','1607','1703','1709','1803','1809','1903','1909','2004','20H2','21H1', '21H2')
+        OrderOfWindowsVersions = @('1507','1511','1607','1703','1709','1803','1809','1903','1909','2004','20H2','21H1', '21H2', '22H2')
```
Comment (sethbrasile) [inline Function.Get-WindowsVersion.ps1:12]: This and line 45-47 are the only impactful changes in this PR

**Review by wmmatt — APPROVED** [review]
LGTM

**Pull request merged**